dom.d.ts

```diff
@@ -0,0 +1,44 @@
+type Nil = null | undefined;
+type Primitive = string | number | bigint | boolean;
+
+type Prop = Nil | Primitive;
```

Comment:
Technically, most props must be `Stringable`, where `Stringable = Prim | X`, where `X` is an object with a custom `toString`. For example, an instance of `URL` is ok, because it stringifies to a proper URL string. But `{}` is not ok, because it would stringify to `[object Object]`, and Prax would throw an exception because that's just stupid. I'm not 100% sure how to "type" this.

Reply:
Also, my apologies for being unclear: props _are_ allowed to be `nil`. In `str.mjs`, `nil` props are ignored. In `str.mjs`, `nil` props are unset / nullified / attributes removed.